package.json

```diff
@@ -103,7 +103,6 @@
     "@bbc/psammead-navigation": "9.1.1",
     "@bbc/psammead-paragraph": "4.0.16",
     "@bbc/psammead-play-button": "3.0.20",
-    "@bbc/psammead-radio-schedule": "5.1.32",
```

Comment:
I assume there will be a follow-up PR to remove the code from psammead and deprecate the package, I think @HarryVerhoef was working on documenting the process?

Reply:
Correct :)